pkg/storage/unified/resource/server.go

```diff
@@ -120,7 +123,10 @@ type ResourceServerOptions struct {
 
 	// Check if a user has access to write folders
 	// When this is nil, no resources can have folders configured
-	WriteAccess WriteAccessHooks
+	WriteHooks WriteAccessHooks
```

Comment:
^^^ leaving for now -- I think this will be part of the provisiong app we are discussing.  This is the part I think needs platform support rather than entirely managed externally